projects/1.19/assets/tanukidecor/tanukidecor/lang/zh_cn.json

```
@@ -0,0 +1,245 @@
+  "block.tanukidecor.elegant_birdcage": "典雅鸟笼",
+  "block.tanukidecor.fancy_vase": "花式花瓶",
+  "block.tanukidecor.firewood": "柴火",
+  "block.tanukidecor.globe": "地球",
+  "block.tanukidecor.green_counter": "绿色柜台",
+  "block.tanukidecor.green_science_pod": "绿色科学吊舱",
+  "block.tanukidecor.green_stall_tarp": "绿色摊位遮篷",
+  "block.tanukidecor.green_table": "绿色桌子",
+  "block.tanukidecor.green_wall_tarp": "绿色挂壁摊位遮篷",
+  "block.tanukidecor.gorgeous_counter": "华丽柜台",
+  "block.tanukidecor.gorgeous_table": "华丽餐桌",
+  "block.tanukidecor.gumball_machine": "糖果机",
+  "block.tanukidecor.handcart": "手推车",
+  "block.tanukidecor.holiday_tree": "圣诞树",
+  "block.tanukidecor.hourglass": "沙漏",
+  "block.tanukidecor.large_fancy_vase": "大号花式花瓶",
+  "block.tanukidecor.large_striped_vase": "大号条纹花瓶",
+  "block.tanukidecor.long_display_case": "长型展示柜",
+  "block.tanukidecor.mayoral_mini_figure": "马约拉尔迷你雕像",
+  "block.tanukidecor.metronome": "打点器",
+  "block.tanukidecor.minimalist_small_table": "简约风小桌子",
+  "block.tanukidecor.minimalist_table": "简约风折叠桌",
+  "block.tanukidecor.narrow_blue_streamer": "小蓝色带灯",
+  "block.tanukidecor.narrow_green_streamer": "小绿色带灯",
+  "block.tanukidecor.narrow_red_streamer": "小红色带灯",
+  "block.tanukidecor.narrow_yellow_streamer": "小黄色带灯",
+  "block.tanukidecor.narrow_string_lights": "小型彩球灯",
+  "block.tanukidecor.newtons_cradle": "牛顿摆",
+  "block.tanukidecor.orange_science_pod": "橙色科学吊舱",
+  "block.tanukidecor.phonograph": "留声机",
+  "block.tanukidecor.piano": "钢琴",
+  "block.tanukidecor.plasma_ball": "等离子球灯",
+  "block.tanukidecor.regal_small_table": "皇家小餐桌子",
+  "block.tanukidecor.red_mushroom_table": "红蘑菇桌子",
+  "block.tanukidecor.red_science_pod": "红色科学吊舱",
+  "block.tanukidecor.red_stall_tarp": "红色摊位遮篷",
+  "block.tanukidecor.red_wall_tarp": "红色挂壁摊位遮篷",
+  "block.tanukidecor.regal_table": "皇家桌子",
+  "block.tanukidecor.ship_in_a_bottle": "瓶中船",
+  "block.tanukidecor.slot_machine": "老虎机",
+  "block.tanukidecor.small_fancy_vase": "小花式花瓶",
+  "block.tanukidecor.small_striped_vase": "小条纹花瓶",
+  "block.tanukidecor.snowglobe": "雪人展示台",
+  "block.tanukidecor.sweets_bookcase": "甜心书柜",
+  "block.tanukidecor.sweets_mini_table": "甜心迷你桌子",
+  "block.tanukidecor.sweets_table": "甜心桌子",
+  "block.tanukidecor.train_set": "玩具小火车套件",
+  "block.tanukidecor.wide_blue_streamer": "大蓝色带灯",
+  "block.tanukidecor.wide_green_streamer": "大绿色带灯",
+  "block.tanukidecor.wide_red_streamer": "大红色带灯",
+  "block.tanukidecor.wide_yellow_streamer": "大黄色带灯",
+  "block.tanukidecor.wide_string_lights": "大型彩球灯",
+  "block.tanukidecor.wooden_block_bookshelf": "木制书架",
+  "block.tanukidecor.wooden_block_table": "木制桌子",
+  "block.tanukidecor.wooden_block_toys": "木制积木玩具",
+  "block.tanukidecor.yellow_stall_tarp": "黄色摊位遮篷",
+  "block.tanukidecor.yellow_wall_tarp": "黄色挂壁摊位遮篷",
+
+  "message.block.tanukidecor.diy_workbench.disabled": "DIY手工制作被禁用",
+
+  "jei.recipe.category.tanukidecor.diy": " DIY工艺装饰",
+
+  "subtitle.block.tanukidecor.alarm_clock.tick": "闹钟的滴答声",

```

```suggestion
  "subtitle.block.tanukidecor.alarm_clock.tick": "闹钟：滴答",
```
字幕格式应与原版保持一致，具体可见https://zh.minecraft.wiki/w/%E5%AD%97%E5%B9%95